src/main/java/org/kiwiproject/dropwizard/poller/config/PollerHealthCheckConfig.java

```diff
@@ -0,0 +1,41 @@
+package org.kiwiproject.dropwizard.poller.config;
+
+import static java.util.Objects.isNull;
+import static org.kiwiproject.dropwizard.poller.health.ClientPollerLatencyBasedHealthCheck.DEFAULT_AVG_LATENCY_WARNING_THRESHOLD_MILLIS;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.util.Duration;
+import io.dropwizard.validation.MaxDuration;
+import io.dropwizard.validation.MinDuration;
+import lombok.Builder;
+import lombok.Value;
+import org.kiwiproject.dropwizard.poller.health.ClientPollerLatencyBasedHealthCheck;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Specifies configuration properties for the various poller health checks. Some configuration properties apply
+ * to only on specific health check.
+ */
+@Value
+public class PollerHealthCheckConfig {
+
+    /**
+     * The average latency for polls above which the latency-based health check will report as unhealthy. Required.
+     * <p>
+     * Default: {@link ClientPollerLatencyBasedHealthCheck#DEFAULT_AVG_LATENCY_WARNING_THRESHOLD_MILLIS} milliseconds
+     *
+     * @see ClientPollerLatencyBasedHealthCheck
+     */
+    @MinDuration(value = 5, unit = TimeUnit.MILLISECONDS)
```

Comment:
5 _millisecond_ average latency seems really low...but not sure what a better minimum should be

Reply:
This is what we have on site, but I'm guessing we overwrite this. Closing comment for now until it becomes an issue.